airflow/ui/src/layouts/Details/DagRunSelect.tsx

```diff
@@ -34,11 +34,13 @@ export const DagRunSelect = forwardRef<HTMLDivElement>((_, ref) => {
   const [searchParams] = useSearchParams();
   const navigate = useNavigate();
 
+  const offset = parseInt(searchParams.get("offset") ?? "0", 10);
+
   const { data, isLoading } = useGridServiceGridData(
     {
       dagId,
-      limit: 14,
-      offset: 0,
+      limit: 25,
```

Comment:
This can be set using config API value.

https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#default-dag-run-display-number

Reply:
We might delete that config value.

Reply:
Ok, I didn't know that. Thanks.